### ISSUE TITLE
Allow custom form controls in form-control-has-label

### ISSUE THREAD

**Issue by jkomusin-vetro** [vue-a11y/eslint-plugin-vuejs-accessibility#373]
The `form-control-has-label` rule currently looks for `input` and `textinput` elements, however custom input components (in our use case, [bootstrap-vue](https://github.com/bootstrap-vue/bootstrap-vue)'s `b-form-input` component) don't seem to be checked for labels.

Is it reasonable to add a check for an implicit wrapping or explicit id-linked label for custom components? Configurable similar to how `label-has-for` can be configured to look for custom label components. I don't think deeper inspection (like this plugin does on `input` to determine eligibility depending on the type of input) is required for custom input types.

**Comment by vhoyer** [comment]
so like, just to confirm, you are saying that we should add like a `controlComponents` option to `form-control-has-label` so that the giving configuration and file results in an error?

```json
{
  "form-control-has-label": ["error", {
    "controlComponents": ["b-form-input"]
  }]
}
```

```vue
<template>
  <div>
    <b-form-input /> <!-- error here because it has not a label linked to it? -->
  </div>
</template>
```

right? or did I miss something?

**Comment by jkomusin-vetro** [comment]
> so like, just to confirm, you are saying that we should add like a `controlComponents` option to `form-control-has-label` so that the giving configuration and file results in an error?
> 
> right? or did I miss something?

Yes totally correct, your example is what I was hoping to be able to do.

**Comment by G-Rath** [comment]
It seems like this was added via #622, so this issue can be closed
